vizro-core/examples/visual-vocabulary/pages/_pages_utils.py

```diff
@@ -11,13 +11,37 @@
 # To disable logging info messages caused by black.format_str: https://github.com/psf/black/issues/2058
 logging.getLogger("blib2to3").setLevel(logging.ERROR)
 
+VIZRO_CODE_TEMPLATE = """
+import vizro.models as vm
+from vizro import Vizro
 
-def make_code_clipboard_from_py_file(filepath: str):
+{example_code}
+
+page = vm.Page(title="{title}", components=[vm.Graph(figure=fig)])
+dashboard = vm.Dashboard(pages=[page])
+Vizro().build(dashboard).run()
+"""
+# TODO HERE: isort? Do like vizro-ai.
```

Comment:
You could use

```py
def _format_and_lint(code_string: str) -> str:
    # Tracking https://github.com/astral-sh/ruff/issues/659 for proper Python API
    # Good example: https://github.com/astral-sh/ruff/issues/8401#issuecomment-1788806462
    # While we wait for the API, we can autoflake and black to process code strings.

    removed_imports = autoflake.fix_code(code_string, remove_all_unused_imports=True)
    # Black doesn't yet have a Python API, so format_str might not work at some point in the future.
    # https://black.readthedocs.io/en/stable/faq.html#does-black-have-an-api
    formatted = black.format_str(removed_imports, mode=black.Mode())
    return formatted
```

Reply:
Done in [9a8e3c7](https://github.com/mckinsey/vizro/pull/824/commits/9a8e3c7c6a42fbfbbfedf1e77da38d1e95cd608d) with the addition of `isort` also since otherwise the code examples looked quite weird.